app/src/main/java/org/kinecosystem/kinit/view/faq/FAQWebFragment.kt

```diff
@@ -39,6 +43,14 @@ class FAQWebFragment : BaseFragment(){
 
         binding.webview.webViewClient = object : WebViewClient() {
             override fun onPageFinished(view: WebView?, url: String?) {
+                try {
+                    model.versionName = context?.packageManager
+                            ?.getPackageInfo(context?.packageName, 0)?.versionName
+                } catch (e: PackageManager.NameNotFoundException) {
+                    Log.e("SupportUtil", "cant get version name " + e.message)
+                }
+                if (view?.url?.contains(model.userRepository.contactUsUrl) == true)
+                    model.setJavaScript(BuildConfig.DEBUG)
```

Comment:
Why not pass the versionName as a parameter to this method (instead of adding it as a member variable of the model)

Reply:
did not liked the dependency on that param.